scripts/dev/241b_vscode_extension_probe.py

```
@@ -0,0 +1,300 @@
+                }
+                for p in self.probes
+            ],
+        }
+
+
+def probe_build_ok() -> ProbeResult:
+    name = "build_ok"
+    if not OUT_JS.exists():
+        # Spróbuj zbudować
+        result = subprocess.run(
+            ["npm", "run", "build"],
```

Ruff (0.15.13)

[error] Starting a process with a partial executable path

(S607)